src/api/app.js

```diff
@@ -46,3 +65,45 @@ app.get('/', function (req, res) {
 
 const port = 3000;
 app.listen(port, () => logger.info(`Start service on port ${port}!`));
+
+function removeEmailIfOlderThan(email, dateTimeBoundary) {
+    return email["datetime"] > dateTimeBoundary;
+}
+
+function filterByEmail(email, to) {
+    let actualToEmail = email["personalizations"][0]["to"][0]["email"];
+    return actualToEmail === to;
+}
+
+function filterBySubject(email, subject) {
```

Comment:
This function represents a predicate not a command, maybe we can find a better suited name for it like `emailContainsSubject(...)`?